ush/python/pygw/src/pygw/timetools.py

```diff
@@ -9,12 +9,15 @@
 
 
 _DATETIME_RE = re.compile(
-    r"(?P<year>\d{4})(-)?(?P<month>\d{2})(-)?(?P<day>\d{2})(T)?(?P<hour>\d{2})?(:)?(?P<minute>\d{2})?(:)?(?P<second>\d{2})?(Z)?")
+    r"(?P<year>\d{4})(-)?(?P<month>\d{2})(-)?(?P<day>\d{2})"
+    r"(T)?(?P<hour>\d{2})?(:)?(?P<minute>\d{2})?(:)?(?P<second>\d{2})?(Z)?")
 
 _TIMEDELTA_HOURS_RE = re.compile(
-    r"(?P<sign>[+-])?((?P<days>\d+)[d])?(T)?((?P<hours>\d+)[H])?((?P<minutes>\d+)[M])?((?P<seconds>\d+)[S])?(Z)?")
+    r"(?P<sign>[+-])?"
+    r"((?P<days>\d+)[d])?(T)?((?P<hours>\d+)[H])?((?P<minutes>\d+)[M])?((?P<seconds>\d+)[S])?(Z)?")
 _TIMEDELTA_TIME_RE = re.compile(
```

Comment:
These aren't new, but while I'm noticing it: these two constants are named `HOURS` and `TIME`, but they both take in hours, minutes, and seconds (in different formats). Are there better names to use (or maybe a list of regex under one variable)?

Reply:
I don't see why not. 
Do you mind if I hold this for another PR where I add some tests for `timetools.py` and then consolidate this?

Reply:
No objection here.